Support hiera_merge on sudoers fragments

anders-larsson opened the pull request.

ghoneycutt commented on manifests/init.pp:
```diff
@@ -82,6 +83,12 @@
+  } else {
+    $hiera_merge_sudoers_bool = $hiera_merge_sudoers
+  }
+
```
This is needed or you could pass a bad type
`validate_bool($hiera_merge_sudoers_bool)`

anders-larsson commented:
@ghoneycutt Thanks for the info. I'll add that check.

anders-larsson commented:
@jwennerberg That seems right. I've updated the pull request with the changes and also removed the commits to fix travis because they've already been merged by pull request #17.
